frontend/src/components/EmployeeModal/StartDate.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import cn from 'classnames';
+import { useFormContext } from 'react-hook-form';
+import { Label, Input } from '../FormElements/FormElements';
+
+
+import styles from './employeemodal.module.css';
+
+const StartDate = () => {
+  const { register } = useFormContext();
+
+  return (
+    <div className={cn(styles.col1, styles.workingHours)}>
+      <Label htmlFor="start">Start Date:</Label>
+      <Input
+        type="date"
```

Comment:
Right now the label & input are not connected, so accessibility wise the label means nothing. When you use "htmlFor", you're connecting the htmlFor to the input's id value, which you have not set here. To connect the label to the input, you need to do one of two things: 

1. Give the input type you have here an id of "start"
2. Give the label some id attribute and in the input, put in aria-labelledby={id_of_label} where id_of_label is the string id you gave to that label.

Reply:
Thank you for the feedback! It would be linked now :)